docs/benchmark.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+NUM_RUNS=20
+AGENT="/var/cfengine/bin/cf-agent"
+BENCHMARK_POLICY="/northern.tech/cfengine/core/examples/complicated_cpu_bound_benchmark.cf"
+
+# Collect only real result from `time` command
+TIMEFORMAT=%R
+
+# Exit on error
+set -e
+
+total=0
+array=()
+for (( i=0; i<NUM_RUNS; i++))
+do
+    time=$(time ($AGENT -KIf $BENCHMARK_POLICY &> /dev/null) 2>&1)
+    total=$(echo "$total + $time" | bc -l)
+    array+=($time)
+    printf "bench %d: %.03f s\n" $i $time
+done
+
+# Sort array of results
+IFS=$'\n' sorted=($(sort <<<"${array[*]}")); unset IFS
+
+echo
+echo "*** RESULTS ***"
+
+# Max time
+max=$(echo "$NUM_RUNS - 1" | bc -l)
```

Comment:
Maybe avoid reliance on `bc`?

```
max=$(($NUM_RUNS-1))
```

Reply:
I was only able to do this on integers, not reals. Am I doing something wrong?